opencti-platform/opencti-graphql/src/config/conf.js

```diff
@@ -355,6 +355,11 @@ export const configureCA = (certificates) => {
 // App
 export const PORT = nconf.get('app:port');
 
+// Playground
+export const PLAYGROUND_INTROSPECTION_DISABLED = booleanConf('app:graphql:playground:force_disabled_introspection', false);
+export const PLAYGROUND_ENABLED = booleanConf('app:graphql:playground:enabled', true);
+export const GRAPHQL_ARMOR_ENABLED = booleanConf('app:graphql:armor_enabled', true);
+
```

Comment:
Do we have environment variables for these configurations?
I also don't see any graphQL variables specified in this doc https://filigran.notion.site/Configuration-a568604c46d84f39a8beae141505572a#afc194900a7f4791868b339a4e1e6b90

Reply:
The doc will be updated after the merge of the PR. Like all configuration variables, use environment variables will be possible.